src/SFA.DAS.AANHub.Application/Common/Validators/CreateMemberCommandValidator.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using FluentValidation;
-using SFA.DAS.AANHub.Application.Common.Commands;
+using SFA.DAS.AANHub.Application.Commands.CreateMember;
 
 namespace SFA.DAS.AANHub.Application.Common.Validators
 {
-    public class CreateMemberCommandValidator : AbstractValidator<BaseMemberCommand>
+    public class CreateMemberCommandValidator : AbstractValidator<CreateMemberCommand>
     {
         public CreateMemberCommandValidator()
```

Comment:
We need a reusable validator that can be included in all the Create commands. To do this, the validator should be on an interface, which is then implemented by each create command. See validator example [here](https://github.com/SkillsFundingAgency/das-roatp-api/blob/fe881635ace7209e614fbca6145be539be0793b1/src/SFA.DAS.Roatp.Application/Common/UkprnValidator.cs#L6)  and the usage example [here](https://github.com/SkillsFundingAgency/das-roatp-api/blob/fe881635ace7209e614fbca6145be539be0793b1/src/SFA.DAS.Roatp.Application/ProviderCourse/Commands/CreateProviderCourse/CreateProviderCourseCommandValidator.cs#L28)

Reply:
I have abstrated the base validation into `IBaseMemberCommand/BaseMemberCommand`, and introduced this to `CreateMemberCommandValidator`